interface.html

```diff
@@ -5,16 +5,19 @@
   </header>
 
   <ul class="nav nav-tabs" role="tablist">
-    <li role="presentation" class="active" id="browse-control"><a href="#browse" aria-controls="browse" role="tab" data-toggle="tab">Select Image</a></li>
-    <li role="presentation" class="{{#unless image}}disabled{{/unless}}" id="image-editor-control"><a href="#image-editor" aria-controls="image-editor" role="tab" data-toggle="tab">Image Editor</a></li>
-    <li role="presentation" class="{{#unless image}}disabled{{/unless}}" id="interactive-control"><a href="#interactive" aria-controls="interactive" role="tab" data-toggle="tab">Interactive</a></li>
+    <li role="presentation" class="active" id="browse-control"><a href="#browse" data-fl-target="browse" aria-controls="browse" role="tab" data-toggle="tab">Select Image</a></li>
```

Comment:
`<li class="active" id="browse-control"><a href="#browse" data-toggle="tab">Select Image</a></li>`

This should be the bare HTML structure you need.